process/phases.md

```diff
@@ -108,12 +108,14 @@ At this point:
 
 Entry requirements:
 
-   * Consensus is reached in amongst Working Group members that the feature
+   * Consensus has been reached amongst Working Group members that the feature
```

Comment:
```suggestion
   * Consensus has been reached amongst the Community Group members that the feature
```

Given the structure of the rest of the document, having the entry requirements reference consensus in the CG makes most sense to me. In addition, we should add a point that consensus should also be reached in the WG in addition to the spec merge. It's a minor correction, but I think this reflects what we currently do better.

Reply:
Consensus within the CG is listed as an entry requirement for the previous phase already. This is about the WG having signed off on it as well, which is a separate step (that is listed to happen in phase 4 above).

Reply:
I was reading "The Community Group has reached consensus in support of the feature" vs. "Consensus has been reached among Community Group members that the feature is complete" as different things. We have been pushing proposals to be complete before a Phase 4 vote, so a minor edit may be to make Phase 4 entry requirements to use stronger language. I don't feel strongly about this, resolving.

Reply:
Ah, I see what you mean, good point. Okay, I made the edit to strengthen the phase 4 entry requirement as you suggested.

The doc explicitly says that phases 4 and 5 happen in the WG and that with voting for phase 4 the CG fully hands off the feature. So a CG-related requirement after this point wouldn't really make sense AFAICS.